tests/apollo/util/bft.py

```diff
@@ -33,7 +33,7 @@
 
 from util.test_base import repeat_test
 
-sys.path.append(os.path.abspath("util/pyclient"))
+sys.path.append(os.path.abspath("pyclient"))
```

Comment:
@shgandhi , probably the more reliable way, i.e. the one non-dependent on where the test is invoked from is to do something like this:
`sys.path.append(os.path.join(os.path.abspath(__file__), "pyclient")) `